src/pages/MissionAndPurpose/MissionAndPurpose.scss

```diff
@@ -0,0 +1,37 @@
+.about-mission-purpose{
+  font-weight: 400; 
+  text-align: left;
+
+  &__body{
+    margin-bottom: 75px;
```

Comment:
Lets use a divisible of 8 for spacing - in this case you could use 72px (although this seems like too much space) could you show design your work and see what they think?